app.go

```diff
@@ -99,7 +99,6 @@ func (a *App) Run() error {
 		}
 	}
 	for _, srv := range a.opts.servers {
-		srv := srv
```

Comment:
```shell
Kratos on  fix/lint [!] via 🐹 v1.23.2 
➜ golangci-lint run
app.go:102:3: The copy of the 'for' variable "srv" can be deleted (Go 1.22+) (copyloopvar)
                srv := srv
                ^

```

fix error by copyloopvar. But this lint need change go version to 1.22 in `Go.Mod`. So i just run in local.